contracts/handlers/ERC20Handler.sol

```diff
@@ -100,14 +138,17 @@ contract ERC20Handler is IDepositExecute, HandlerHelpers, ERC20Safe {
         );
     }
 
-    // execute a deposit
-    // bytes memory data passed into the function should be constructed as follows:
-
-    // resourceID                             bytes32     bytes  0 - 32
-    // amount                                 uint256     bytes   32 - 64
-    // --------------------------------------------------------------------
-    // destinationRecipientAddress length     uint256     bytes  64 - 96
-    // destinationRecipientAddress            bytes       bytes  96 - END
+    /**
+        @notice Deposit execution should be initiated when a proposal is executed in the Bridge contract.
```

Comment:
Lets avoid the word deposit here, deposit should refer to the initiation step, proposal execution should be the completion step

Reply:
```suggestion
        @notice Proposal execution should be initiated when a proposal is finalized in the Bridge contract.
```